src/styles/upload-collection.scss

```
@@ -53,8 +54,25 @@ $fd-upload-collection-button-group-small-margin: 0.5rem !default;
     padding-top: 0.625rem;
   }
 
-  &__status-group {
+  .#{$block}__status-group {
+    @include fd-flex();
+
     margin-top: $fd-upload-collection-side-margin;
+
+    &-item {
+      @include fd-set-margins-x-equal(0.25rem);
+
+      flex-shrink: 1;
+      min-width: 0;
+
+      &:first-child {
+        @include fd-set-margins-x(0, 0.25rem);
+      }
+
+      &-text {
+        @include fd-ellipsis();
```

```suggestion
        @include fd-ellipsis();
        line-height: initial;
```
This should fix that cut-down characters